src/core/Tools.cpp

```diff
@@ -165,6 +168,12 @@ namespace Tools
         return QObject::tr("%1 minute(s)", nullptr, minutes).arg(minutes);
     }
 
+    QString toString(const QDateTime& dateTime)
+    {
+        static QLocale locale;
+        return locale.toString(dateTime, QLocale::ShortFormat);
+    }
```

Comment:
This function name is too generic for it's purpose. Either add this to core/clock.h|cpp or name it something like datetimeToString(...)